src/DevBetterWeb.Core/Entities/Invitation.cs

```diff
@@ -9,12 +10,27 @@ public class Invitation : BaseEntity, IAggregateRoot
     public string InviteCode { get; private set; }
     public string PaymentHandlerSubscriptionId { get; private set; }
     public bool Active { get; private set; } = true;
+    public DateTime DateCreated { get; private set; }
+    public DateTime DateOfUserPing { get; private set; } = DateTime.MinValue;
+    public DateTime DateOfLastAdminPing { get; private set; } = DateTime.MinValue;
 
     public Invitation(string email, string inviteCode, string paymentHandlerSubscriptionId)
     {
       Email = email;
       InviteCode = inviteCode;
       PaymentHandlerSubscriptionId = paymentHandlerSubscriptionId;
+      DateCreated = DateTime.Today;
+    }
+
+    // for testing
+    public Invitation(string email, string inviteCode, string paymentHandlerSubscriptionId, DateTime dateCreated, DateTime dateOfUserPing, DateTime dateOfLastAdminPing)
+    {
+      Email = email;
+      InviteCode = inviteCode;
+      PaymentHandlerSubscriptionId = paymentHandlerSubscriptionId;
+      DateCreated = dateCreated;
+      DateOfUserPing = dateOfUserPing;
+      DateOfLastAdminPing = dateOfLastAdminPing;
     }
 
     private Invitation()
```

Comment:
I think this could be empty (or perhaps eliminated) if all of these sets were done where the properties are defined.

It's generally preferable to set property defaults inline since it's easier to see whether you forgot to do so with any of them without the vertical separation between defining them and initializing them in the constructor.

Reply:
these aren't defaults, they're generally test values. Defaults _are_ set where the properties are defined

Reply:
I was referencing the contents of the `private Invitation()` constructor, which just has things set to "" and such.